base/xx-cc

```diff
@@ -8,15 +8,28 @@ if [ -n "$XX_MIRROR" ]; then
   mirrors="$XX_MIRROR"
 fi
 
+_wget() {
+  if ! wget "$@"; then
+    # if it fails install GNU wget in case of limitation
+    # with busybox wget
+    installwget
```

Comment:
is there a check we could put in here to validate that we don't have gnu-wget already?

Reply:
Done. A bit hacky but works fine.